src/styles/components/_component-grid.scss

```
@@ -0,0 +1,75 @@
+$breakpoint-tablet: 820px;
+
+.components {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1rem;
+
+  &-grid {
+    &__item {
+      display: flex;
+
```

I know you and I talk about this, but I think it just got missed. There's something responsive issues with grid again, so I just wanted to pass along the fixes we talked about: 
```suggestion
..components {
  display: grid;
  grid-template-columns: 1fr;
  gap: 1rem;

  &-grid {
    width: 100%;
    padding: 0;
  }

  &-grid__item {
    width: 100%;
  }

  &-grid {
    &__item {
      display: flex;
      justify-content: center;

      &--accordion {
        grid-area: accordion;
      }

      &--dialog {
        grid-area: dialog;

```

---

Perfect! I added this and it helped center the layout :)